js/extension/components/FeatureReports.jsx

```diff
@@ -4,6 +4,7 @@ import PropTypes from "prop-types";
 import React from "react";
 import { Button, Col, Collapse, Glyphicon, Panel, Row } from "react-bootstrap";
 import Select from "react-select";
+import { default as html2pdf } from "html2pdf.js";
```

Comment:
it didn't work for me at first, I had to run npm install to update the package-lock.json so I committed and pushed it